test/test_autograd.py

```diff
@@ -7564,6 +7588,25 @@ def test_non_differentiable(self):
             # No differentiable outputs, shouldn't error
             eq = foo == bar
 
+    def test_create_new_zeros_with_same_meta(self):
+        new_zeroes_fn = torch.ops.aten._new_zeros_with_same_feature_meta
+
+        def assert_same_meta(a, b):
+            # Layout is the same
+            self.assertEqual(a.size(), b.size())
+            self.assertEqual(a.stride(), b.stride())
+            self.assertEqual(a.storage_offset(), b.storage_offset())
+            self.assertEqual(len(a.storage()), len(b.storage()))
+
+            # TensorOptions is same
+            self.assertEqual(a.dtype, b.dtype)
+
+        a = torch.randn(5, dtype=torch.float).resize_(4)[1:]
```

Comment:
nit: narrow is better than resize here.

Reply:
I was thinking of using resize here to make sure that the implementation of our new zeros actually accesses the storage numel, and doesn't just rely on the numel of the base (which is accessible if we just do a view like narrow).

Reply:
You can detach to remove the base.
But ok.